launch/caret.launch.py

```diff
@@ -38,7 +38,7 @@ def generate_launch_description():
 						"ros2_caret:*executor",
 						"ros2_caret:dds_bind*",
 						]
-		if os.environ["ROS_DISTRO"] in ["iron", "rolling"]:
+		if os.environ["ROS_DISTRO"][0] >= 'i':
```

Comment:
```suggestion
		if os.environ["ROS_DISTRO"][0] >= "i":
```

Maybe we should change it for consistency.